test/kad-dht.spec.js

```diff
@@ -187,10 +193,10 @@ describe('KadDHT', () => {
         (cb) => dhtB.swarm.dial(peerA.id, cb)
       ], done)
     })
-  })
+  }).timeout(30 * 1000)
 
   // Might need to disable on ci
-  it('find peer query', (done) => {
+  it.skip('find peer query', (done) => {
```

Comment:
Why disable this test?

Reply:
It just takes too long to run on CI.  See the comment above which was pre-existing